core/tabs/utils/create-bootable-usb.sh

```diff
@@ -66,6 +66,11 @@ fetch_debian_latest_iso() {
     printf "%b\n" "${GREEN} Selected Debian Linux (latest) ISO URL: ${RC} $DEBIAN_URL"
 }
 
+fetch_fedora_latest_iso() {
+    FEDORA_URL=$(curl -s 'https://www.fedoraproject.org/releases.json' | jq -r '.[] | select(.version == "40" and .arch == "x86_64" and .variant == "Workstation") | select(.link | match(".*Live-x86_64*")) | .link')
```

Comment:
Tested your code, works as expected **but** jquery is not POSIX-compliant and not all distros ship it by default. This script could install it before running by a think this is a better idea:

```suggestion
    FEDORA_URL=$(curl -s 'https://www.fedoraproject.org/releases.json' |
    grep -o '"link": "[^"]*' |
    sed -e 's/"link": "//' |
    awk -v version="40" -v arch="x86_64" -v variant="Workstation" ' BEGIN { FS="," }
    {
        if ($0 ~ version && $0 ~ arch && $0 ~ variant && $0 ~ "Live-x86_64") {
            print $0
        }
    }' |
    head -n 1)
```
And please, fix my tabs and spaces here, I'm typing this from my phone.

Reply:
jq is a JSON parser, jQuery is a JS library. They are not the same thing. All of these are external tools (even curl, grep, sed, which are practically universal), though it is obviously preferable to use more common programs whenever possible.